Bearded.UI/NullableAttributes.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+
+namespace Bearded.UI
+{
+    // To deal with special cases with non nullable reference types, the C# compiler team has made some attributes,
+    // which they then only added for .NET Core. So I've had to add my own copy here.
```

Comment:
This is fair enough.

This should be fixed when we migrate this library to OpenTK 4 stable.

Suggestion: mark this as Obsolete to make sure we don't forget to replace this with the .NET Core one. Feel free to file an issue too.

Reply:
Issue created: https://github.com/beardgame/ui/issues/20